src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV1Spec.scala

```diff
@@ -38,6 +38,32 @@ class UserRoutesV1Spec extends UserRoutesSpecHelper{
     }
   }
 
+  it should "create a user if ToS is enabled and the user specifies the ToS parameter correctly" in withTosEnabledRoutes { samRoutes =>
+    Post("/register/user/v1?tos=app.terra.bio/#terms-of-service") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Created
+      val res = responseAs[UserStatus]
+      res.userInfo.userSubjectId.value.length shouldBe 21
+      res.userInfo.userEmail shouldBe defaultUserEmail
+      res.enabled shouldBe Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true)
+    }
+
+    Post("/register/user/v1?tos=app.terra.bio/#terms-of-service") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Conflict
+    }
+  }
+
+  it should "forbid the registration if ToS is enabled and the user doesn't specify a ToS parameter" in withTosEnabledRoutes { samRoutes =>
+    Post("/register/user/v1") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Forbidden
```

Comment:
I suggest adding `responseAs[ErrorReport].message should startWith "You must accept the Terms of Service in order to register."` or similar here, to distinguish between a Forbidden thrown by the TOS check and a forbidden thrown by something else. This applies to all your new tests that look for Forbidden.